cli/man/transact-workload.1.md

```diff
@@ -66,8 +66,9 @@ OPTIONS
   of submitting the HTTP requests. (default: `30`)
 
 `--workload WORKLOAD `
-: The workload to be submitted. The possibly value is `smallbank`. The smallbank
-  workload submits batches containing sabre Transactions for smallbank payloads.
+: The workload to be submitted. The possible values are `smallbank` or `command`.
+  The smallbank workload submits batches containing sabre Transactions for
+  smallbank payloads.
```

Comment:
Probably worthwhile to make the last sentence include that the `command` option does the same. Perhaps something like `Determines the type of sabre transactions contained within the batches submitted by the workload, either smallbank or command payloads.`

Reply:
Updated